# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1649437325 -7200
#      Wed Mar 02 21:53:04 2022 +0100
# Node ID dfeeafd9ae10d4467a9c91c82c891e7833a0e68b
# Parent  c8f0575b6adb67d0dd651da2105b1e067c6849e9
Misc: Report missing Fluent strings.

diff --git a/intl/l10n/Localization.h b/intl/l10n/Localization.h
--- a/intl/l10n/Localization.h
+++ b/intl/l10n/Localization.h
@@ -46,17 +46,17 @@ namespace intl {
     const nsTArray<nsCString>& aErrors, ErrorResult& aRv,
     nsIGlobalObject* aGlobal) {
   if (!aErrors.IsEmpty()) {
     if (xpc::IsInAutomation()) {
       aRv.ThrowInvalidStateError(aErrors.ElementAt(0));
       return true;
     }
 
-#if defined(NIGHTLY_BUILD) || defined(MOZ_DEV_EDITION) || defined(DEBUG)
+#if 1
     dom::Document* doc = nullptr;
     if (aGlobal) {
       nsPIDOMWindowInner* innerWindow = aGlobal->AsInnerWindow();
       if (innerWindow) {
         doc = innerWindow->GetExtantDoc();
       }
     }
 
